src/libraries/System.Security.Cryptography/tests/AesCcmTests.cs

```diff
@@ -12,7 +12,6 @@ namespace System.Security.Cryptography.Tests
     public class AesCcmTests : CommonAEADTests
     {
         [Theory]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51332", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
This is for CCM, not GCM. I would not expect these tests to pass on mobile apple crypto.

Reply:
Correct, but they are not running because of the `[ConditionalClass(typeof(AesCcm), nameof(AesCcm.IsSupported))]`

I removed the ActiveIssue since it was closed and the tests are correctly skipped.